components/automatic-grid/script.js

```diff
@@ -191,6 +198,9 @@ class AutomaticGrid extends HTMLElement {
 
         this.dropItem(event.target);
 
+        //Reset the registry point to the top left
+        this._draggedItem.style.transform = `translateX(0) translateY(0)`;
```

Comment:
After the first time the component-slot is dragged, this doesn't help much on the second drag for the element to not move beneath the cursor.
We can just reset the whole **style** attribute.

`this._draggedItem.style = '';`

This^ will place it according to the flex layout once more.
If we intend to make stick to the position it is dropped, we should figure out why on the second `dragStart` the `this._draggedItem.getBoundingClientRect().x` returns a different value compared against the first `dragStart` although the element looks like it is in the same place.